app/src/main/java/org/simple/clinic/drugs/search/DrugSearchListItem.kt

```diff
@@ -75,6 +99,7 @@ sealed class DrugSearchListItem : PagingItemAdapter.Item<DrugSearchListItem.Drug
       return when {
         oldItem is DrugSearchResult && newItem is DrugSearchResult -> oldItem.drug.id == newItem.drug.id
         oldItem is Divider && newItem is Divider -> false
+        oldItem is NewCustomDrug && newItem is NewCustomDrug -> true
```

Comment:
Do we need to compare the names of the new custom drug in this case to see if they refer to the same item?

Reply:
The UI transition were not smooth when the name changes. So left it as true to prevent the item from updating, we anyway get the search query from the edit text directly. So we don't need to worry about item data not getting updated.